Python: PR #2900 breaks system tests

E.g., from [this Kokoro job](https://source.cloud.google.com/results/invocations/db8f1aa0-2f95-4f0a-a792-c1e56653af46/targets/cloud-devrel%2Fclient-libraries%2Fgoogle-cloud-python%2Fpresubmit%2Fvision/log):

```python
_ ERROR at setup of TestVisionClientVpcsc.test_batch_annotate_images_read_blocked _
    def setUpModule():
>       VisionSystemTestBase.client = vision.ImageAnnotatorClient()
tests/system.py:55:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
google/cloud/vision_v1/gapic/image_annotator_client.py:164: in __init__
    address=api_endpoint, channel=channel, credentials=credentials
google/cloud/vision_v1/gapic/transports/image_annotator_grpc_transport.py:70: in __init__
    "grpc.max_receive_message_length": -1,
google/cloud/vision_v1/gapic/transports/image_annotator_grpc_transport.py:109: in create_channel
    address, credentials=credentials, scopes=cls._OAUTH_SCOPES, **kwargs
../api_core/google/api_core/grpc_helpers.py:206: in create_channel
    return grpc.secure_channel(target, composite_credentials, **kwargs)
.nox/system-3-7/lib/python3.7/site-packages/grpc/__init__.py:1782: in secure_channel
    credentials._credentials, compression)
.nox/system-3-7/lib/python3.7/site-packages/grpc/_channel.py:1021: in __init__
    credentials)
src/python/grpcio/grpc/_cython/_cygrpc/channel.pyx.pxi:429: in grpc._cython.cygrpc.Channel.__cinit__
    ???
src/python/grpcio/grpc/_cython/_cygrpc/arguments.pyx.pxi:78: in grpc._cython.cygrpc._ChannelArgs.__cinit__
    ???
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
>   ???
E   ValueError: too many values to unpack (expected 2)
src/python/grpcio/grpc/_cython/_cygrpc/arguments.pyx.pxi:37: ValueError
___ ERROR at setup of TestVisionClientVpcsc.test_import_product_sets_blocked ___
    def setUpModule():
>       VisionSystemTestBase.client = vision.ImageAnnotatorClient()
tests/system.py:55:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
google/cloud/vision_v1/gapic/image_annotator_client.py:164: in __init__
    address=api_endpoint, channel=channel, credentials=credentials
google/cloud/vision_v1/gapic/transports/image_annotator_grpc_transport.py:70: in __init__
    "grpc.max_receive_message_length": -1,
google/cloud/vision_v1/gapic/transports/image_annotator_grpc_transport.py:109: in create_channel
    address, credentials=credentials, scopes=cls._OAUTH_SCOPES, **kwargs
../api_core/google/api_core/grpc_helpers.py:206: in create_channel
    return grpc.secure_channel(target, composite_credentials, **kwargs)
.nox/system-3-7/lib/python3.7/site-packages/grpc/__init__.py:1782: in secure_channel
    credentials._credentials, compression)
.nox/system-3-7/lib/python3.7/site-packages/grpc/_channel.py:1021: in __init__
    credentials)
src/python/grpcio/grpc/_cython/_cygrpc/channel.pyx.pxi:429: in grpc._cython.cygrpc.Channel.__cinit__
    ???
src/python/grpcio/grpc/_cython/_cygrpc/arguments.pyx.pxi:78: in grpc._cython.cygrpc._ChannelArgs.__cinit__
    ???
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
>   ???
E   ValueError: too many values to unpack (expected 2)
```

--- a/src/main/resources/com/google/api/codegen/py/transport.snip
+++ b/src/main/resources/com/google/api/codegen/py/transport.snip
@@ -53,7 +53,7 @@
                     options={
                         'grpc.max_send_message_length': -1,
                         'grpc.max_receive_message_length': -1,
-                    },
+                    }.items(),
                 )
 
             self._channel = channel

--- a/src/test/java/com/google/api/codegen/gapic/testdata/py/python_library.baseline
+++ b/src/test/java/com/google/api/codegen/gapic/testdata/py/python_library.baseline
@@ -4279,7 +4279,7 @@ class LibraryServiceGrpcTransport(object):
                 options={
                     'grpc.max_send_message_length': -1,
                     'grpc.max_receive_message_length': -1,
-                },
+                }.items(),
             )
 
         self._channel = channel
@@ -4783,7 +4783,7 @@ class MyProtoGrpcTransport(object):
                 options={
                     'grpc.max_send_message_length': -1,
                     'grpc.max_receive_message_length': -1,
-                },
+                }.items(),
             )
 
         self._channel = channel

--- a/src/test/java/com/google/api/codegen/gapic/testdata/py/python_multiple_services.baseline
+++ b/src/test/java/com/google/api/codegen/gapic/testdata/py/python_multiple_services.baseline
@@ -1432,7 +1432,7 @@ class DecrementerServiceGrpcTransport(object):
                 options={
                     'grpc.max_send_message_length': -1,
                     'grpc.max_receive_message_length': -1,
-                },
+                }.items(),
             )
 
         self._channel = channel
@@ -1560,7 +1560,7 @@ class IncrementerServiceGrpcTransport(object):
                 options={
                     'grpc.max_send_message_length': -1,
                     'grpc.max_receive_message_length': -1,
-                },
+                }.items(),
             )
 
         self._channel = channel

--- a/src/test/java/com/google/api/codegen/gapic/testdata/py/python_no_path_templates.baseline
+++ b/src/test/java/com/google/api/codegen/gapic/testdata/py/python_no_path_templates.baseline
@@ -1121,7 +1121,7 @@ class NoTemplatesApiServiceGrpcTransport(object):
                 options={
                     'grpc.max_send_message_length': -1,
                     'grpc.max_receive_message_length': -1,
-                },
+                }.items(),
             )
 
         self._channel = channel

--- a/src/test/java/com/google/api/codegen/gapic/testdata/py/python_samplegen_config_migration_library.baseline
+++ b/src/test/java/com/google/api/codegen/gapic/testdata/py/python_samplegen_config_migration_library.baseline
@@ -4270,7 +4270,7 @@ class LibraryServiceGrpcTransport(object):
                 options={
                     'grpc.max_send_message_length': -1,
                     'grpc.max_receive_message_length': -1,
-                },
+                }.items(),
             )
 
         self._channel = channel
@@ -4774,7 +4774,7 @@ class MyProtoGrpcTransport(object):
                 options={
                     'grpc.max_send_message_length': -1,
                     'grpc.max_receive_message_length': -1,
-                },
+                }.items(),
             )
 
         self._channel = channel

--- a/src/test/java/com/google/api/codegen/protoannotations/testdata/python_library.baseline
+++ b/src/test/java/com/google/api/codegen/protoannotations/testdata/python_library.baseline
@@ -4272,7 +4272,7 @@ class LibraryServiceGrpcTransport(object):
                 options={
                     'grpc.max_send_message_length': -1,
                     'grpc.max_receive_message_length': -1,
-                },
+                }.items(),
             )
 
         self._channel = channel
@@ -4776,7 +4776,7 @@ class MyProtoGrpcTransport(object):
                 options={
                     'grpc.max_send_message_length': -1,
                     'grpc.max_receive_message_length': -1,
-                },
+                }.items(),
             )
 
         self._channel = channel

--- a/src/test/java/com/google/api/codegen/protoannotations/testdata/python_library_no_gapic_config.baseline
+++ b/src/test/java/com/google/api/codegen/protoannotations/testdata/python_library_no_gapic_config.baseline
@@ -3781,7 +3781,7 @@ class LibraryServiceGrpcTransport(object):
                 options={
                     'grpc.max_send_message_length': -1,
                     'grpc.max_receive_message_length': -1,
-                },
+                }.items(),
             )
 
         self._channel = channel
@@ -4280,7 +4280,7 @@ class MyProtoGrpcTransport(object):
                 options={
                     'grpc.max_send_message_length': -1,
                     'grpc.max_receive_message_length': -1,
-                },
+                }.items(),
             )
 
         self._channel = channel

